Document/Testcases/0x00b_OMTG-DATAST_iOS.md

```diff
@@ -33,15 +82,83 @@ Use a define to enable NSLog statements for development and debugging, and disab
+* [Android KeyStore][19149717]
+
+
+## <a name="OMTG-DATAST-003"></a>OMTG-DATAST-003: Test that no sensitive data leaks to cloud storage
+
+### Black-box Testing
+
+### White-box Testing
+
+### Remediation
+
+### References
+
+
+## <a name="OMTG-DATAST-004"></a>OMTG-DATAST-004: Test for sending sensitvie data to 3rd Parties
+
+### Black-box Testing
+
+### White-box Testing
+
+### Remediation
+
+### References
+
+
+## <a name="OMTG-DATAST-005"></a>OMTG-DATAST-005: Test that keyboard cache is disabled for sensitive data
+
+
+### Black-box Testing
+
+Reset your iOS device keyboard cache by going through:
+
+Settings > General > Reset > Reset Keyboard Dictionary
+
+Proceed to use the application's functionalities. Identify the functions which allow users to enter sensitive data.
+
+Dump the keyboard cache file dynamic-text.dat at the following directory:
+/private/var/mobile/Library/Keyboard/
+
+Look for sensitive data such as username, email addresses, credit card numbers, etc.
+If the sensitive data can be obtained through the keyboard cache file, it fails this test.
+
+### White-box Testing
+
+[Describe how to test for this issue using static and dynamic analysis techniques. This can include everything from simply monitoring aspects of the app’s behavior to code injection, debugging, instrumentation, etc. ]
+
+### Remediation
+
+The application must ensure that data typed into text fields which contains sensitive information must not be cached. This can be achieved by disabling the feature programmatically by using the AutoCorrection = FALSE directive in the desired UITextFields. For data that should be masked such as PIN and passwords, set the textField.secureTextEntry to YES.
+
+### References
+
 - [link to relevant how-tos, papers, etc.]
 
 
+## <a name="OMTG-DATAST-006"></a>OMTG-DATAST-006: Test that clipboard is deactivated for sensitive input fields
+
+## <a name="OMTG-DATAST-007"></a>OMTG-DATAST-007: Test that no sensitive data is exposed via IPC mechanisms
+
+## <a name="OMTG-DATAST-008"></a>OMTG-DATAST-008: Test that no sensitive data is exposed via the user interface or screenshots
+
+## <a name="OMTG-DATAST-009"></a>OMTG-DATAST-009: Test for Sensitive Data in Backups
+
 ## <a name="OMTG-DATAST-010"></a>OMTG-DATAST-010: Test that no sensitive data leaks when backgrounded
 
 
 ### Black-box Testing
 
-[Describe how to test for this issue using static and dynamic analysis techniques. This can include everything from simply monitoring aspects of the app’s behavior to code injection, debugging, instrumentation, etc. ]
+Proceed to a page on the application which displays sensitive information such as username, email address, account details, etc. Background the application by hitting the Home button on your iOS device. SSH into your iOS device and proceed to the following directory:
```

Comment:
Again there are different methods to connect to your iOS device. See comment above